client/extensions/woocommerce/state/sites/settings/email/test/actions.js

```diff
@@ -0,0 +1,141 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { requestSettings, submitMailChimpApiKey } from '../actions';
+import useNock from 'test/helpers/use-nock';
+import {
+	WOOCOMMERCE_MAILCHIMP_SETTINGS_REQUEST,
+	WOOCOMMERCE_MAILCHIMP_SETTINGS_REQUEST_SUCCESS,
+	WOOCOMMERCE_MAILCHIMP_API_KEY_SUBMIT,
+	WOOCOMMERCE_MAILCHIMP_API_KEY_SUBMIT_SUCCESS,
+} from 'woocommerce/state/action-types';
+
```

Comment:
Missing tests for store info, campaign defaults and newsletter settings?

Reply:
missing a lot of more tests - working on it.